README.md

```diff
@@ -0,0 +1,62 @@
+# Workflow Library Manager Plugin
+
+## Summary
+
+The plugin provides a primitive management of Workflow library scripts.
+
+**Warning!** The project is a **Proof of concept**. The functionality may change before the release (if it happens, of course). Use it on your own risk.
+
+Supported features:
+* Library loading from Git and Github (requires an installed Git plugin)
+* Multiple library Loading from a local file
+
+## Usage
+
+The plugin adds a global `fileLoader` variable, which can be used to invoke the Workflow library loading from multiple sources. 
```

Comment:
It sounds (from reading below) that this is more than just a variable, no? It provides `fromGit` and `load` functions at least, right? Might make sense to describe it as a DSL (ala the `docker` DSL).